src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -79,7 +79,7 @@
     {/if}
   </div>
   <div
-    class="bg-[#DDD6FE] bg-contain bg-no-repeat xl:flex xl:min-h-full xl:flex-1 xl:items-stretch xl:justify-center"
+    class="bg-[#DDD6FE] bg-contain bg-no-repeat xl:flex xl:min-h-full xl:max-w-[1050px] xl:flex-1 xl:items-stretch xl:justify-center"
```

Comment:
What is the 1050 magic number here? 1/2 of the total screen width of xl?

I kind of like static max-w's because it keeps the width stationary when it hits max instead of always expanding/minimizing so I think I prefer this to like `max-w-[50dvw]`

Nothing to do thinking out loud

Reply:
I just set it to the max width of the image before it started showing the gap.

Reply:
Don't think that will work with the `background-image` (which is what we're using here).

Reply:
I wonder if `bg-[100%_auto]` would be better here

Reply:
True, I just figured it would be less brittle than a magic number